packages/http-client/src/index.test.ts

```diff
@@ -45,8 +45,14 @@ describe("getProjectDetails", () => {
       apiUrl,
       projectId: notPublishedProjectId,
     });
-    expect(response.toString()).toBe(
-      `Project ${notPublishedProjectId} not found or not published yet. Please contact us to get help.`
-    );
+    if (response instanceof Error) {
```

Comment:
how can response be an error? it would be possible if you return an Error instance, but in loadProject you throw an error ... what am I missing?